head/src/test/kotlin/io/qalipsis/core/head/model/converter/CampaignConfigurationConverterImplTest.kt

```diff
@@ -38,14 +54,8 @@ internal class CampaignConfigurationConverterImplTest {
     @InjectMockKs
     private lateinit var converter: CampaignConfigurationConverterImpl
 
-
     @RelaxedMockK
-    private lateinit var headConfiguration: HeadConfiguration
-
-    @BeforeEach
-    internal fun setup() {
-        every { headConfiguration.zones } returns setOf(Zone(key = "FR", title = "France", description = "description"), Zone(key = "EN", title = "England", description = "description"))
-    }
+    private lateinit var hooks: List<CampaignHook>
```

Comment:
You should rather create several mocks from CampaignHook and inject them manually into the constructor of CampaignConfigurationConverterImpl.

Also, please verifiy that the hooks are executed in the order you provide, with the expected arguments.

Reply:
This point is still not solved to me.